src/lifx/light.js

```diff
@@ -134,7 +134,7 @@ Light.prototype.colorRgbHex = function(hexString, duration, callback) {
 Light.prototype.maxIR = function(brightness, callback) {
   validate.irBrightness(brightness, 'light setMaxIR method');
 
-  brightness = Math.round(brightness / constants.IR_MAXIMUM_BRIGHTNESS * 65535);
+  brightness = Math.round((brightness / constants.IR_MAXIMUM_BRIGHTNESS) * 65535);
```

Comment:
Revert these unrelated changes.

Reply:
I revert this changes, but it drives me crazy.... infact nobody normal knows the order of execution of the operators * /.